docs/src/pages/[platform]/components/breadcrumbs/examples/NextBreadcrumbsExample.tsx

```diff
@@ -0,0 +1,38 @@
+import Link from 'next/link';
+import { Breadcrumbs } from '@aws-amplify/ui-react';
+import { useRouter } from 'next/router';
+
+export default function NextBreadcrumbsExample() {
+  const { asPath } = useRouter();
+
+  const nestedRoutes = asPath
+    .split('#')[0]
```

Comment:
Why not use https://developer.mozilla.org/en-US/docs/Web/API/URL to get [pathname](https://developer.mozilla.org/en-US/docs/Web/API/URL/pathname)?